aries-site/src/components/seo/Meta.js

```diff
@@ -17,9 +17,9 @@ export const Meta = ({
   const segment = 'corporate';
   const lifecycle = 'support';
   const pageContent = 'products';
-  const csp = `default-src 'self' 'unsafe-eval'; 
+  const csp = `default-src 'self' 'unsafe-eval';
   style-src 'self' *.hpe.com/hfws-static/5/css/ 'unsafe-inline';
-  connect-src 'self' *.githubusercontent.com/grommet/hpe-design-system/ https://api.github.com/repos/grommet/hpe-design-system/pulls https://api.github.com/repos/grommet/hpe-design-system/commits https://www.google-analytics.com https://www.github.com/grommet/ https://eyes.applitools.com *.hpe.com/hpe/api/ https://iad1.qualtrics.com/API/v3/surveys/ https://api.spacexdata.com/;
+  connect-src 'self' *.githubusercontent.com/grommet/hpe-design-system/ https://api.github.com/repos/grommet/hpe-design-system/pulls https://api.github.com/repos/grommet/hpe-design-system/commits https://www.google-analytics.com https://www.github.com/grommet/ https://eyes.applitools.com *.hpe.com/hpe/api/ https://iad1.qualtrics.com/API/v3/surveys/ https://api.spacexdata.com/ https://raw.githubusercontent.com;
```

Comment:
Wondering if we should make the url more specific to what we need so instead of `https://raw.githubusercontent.com`, something like `https://raw.githubusercontent.com/.../...`